js/utils/templateHelpers.js

```diff
@@ -4,11 +4,17 @@ import {
   isHiRez, isLargeWidth, isSmallHeight, getAvatarBgImage,
 } from './responsive';
 import { upToFixed } from './number';
+import twemoji from 'twemoji';
 
 export function polyT(...args) {
   return app.polyglot.t(...args);
 }
 
+export function parseEmojis(msg) {
```

Comment:
Minor, but can we change the argument to 'text'. 'msg' was appropriate in the chat view because it was parsing a chat message, but since we've added this to be used generically, I think the argument name should follow suit.